mobile/ios/ios-core.ts

```diff
@@ -980,30 +983,80 @@ export class PlistService implements Mobile.IiOSDeviceSocket {
 	}
 }
 
-export class GDBServer implements Mobile.IGDBServer {
-	private socket: Mobile.IiOSDeviceSocket  = null;
+util.inherits(GDBStandardOutputAdapter, stream.Transform);
+function GDBStandardOutputAdapter(opts?: stream.TransformOptions) {
```

Comment:
Can we do this with class - for example `class GDBStandardOutputAdapter extends stream.Transform`?